stages/org.osbuild.mkfs.fat

```diff
@@ -58,16 +74,19 @@ def main(devices, options):
     volid = options["volid"]
     label = options.get("label")
     fatsize = options.get("fat-size")
-    opts = []
+    geometry = options.get("geometry")
+    opts = ["mkfs.fat", "-I", "-i", volid]
```

Comment:
Nitpick: the `opts` no longer holds only options. It would be better to rename it to `cmd` or something similar.